packages/manager/modules/pci/src/projects/project/failover-ips/failover-ips.html

```diff
@@ -0,0 +1,28 @@
+<div data-ui-view>
+    <oui-page-header heading="{{ ::'pci_projects_project_failoverip_title' | translate }}"></oui-page-header>
+    <div class="px-5">
+        <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+        <oui-datagrid rows-loader="$ctrl.loadPartialData($config)" page-size="5">
+            <extra-top>
+                <a class="oui-button oui-button_secondary oui-button_icon-left mb-3" data-ui-sref="pci.projects.project.failover-ips.imports">
```

Comment:
Could be in a `resolve` on state :)

Reply:
Will be done with lazyloading & components :)